db/schema.rb

```diff
@@ -71,7 +71,7 @@
     t.string   "overview_content_type"
     t.integer  "overview_file_size"
     t.datetime "overview_updated_at"
-    t.integer  "order"
+    t.integer  "order",                          default: 1
```

Comment:
@jasonoverby You want to try to avoid committing stuff like this along with the code your working on, as it can make for some nasty merge-conflicts. In general, migrations and schema changes should be committed together, separate from other code changes.